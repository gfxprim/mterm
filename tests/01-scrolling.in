1 4
abcdef
