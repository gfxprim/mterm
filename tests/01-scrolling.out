 -
|d|
|e|
|f|
| |
 -
size 4x1 cursor 3x0
